sasl2-sys/build.rs

```diff
@@ -113,17 +115,20 @@ fn build_sasl(metadata: &Metadata) {
         "--with-pic".into(),
         format!("CPPFLAGS={}", cppflags),
         format!("CFLAGS={}", cflags),
+        format!("CC={}", compiler_path.to_str().unwrap_or_default()),
     ];
     if metadata.target.contains("darwin") {
         configure_args.push("--disable-macos-framework".into());
     }
     if metadata.host != metadata.target {
+        configure_args.push(format!("--build={}", metadata.host));
```

Comment:
Why does this need to be specified? Doesn't `--build` always default to the platform on which you're running `configure`?